### PR TITLE
Adds VAction Unit tests

### PR DESCRIPTION
**Pull request by ontoneio**
- Cleaned up and amended my history a bit to not include all that extra git fluff

### CODE REVIEW

File: src/components/VAction/VAction.vue [hunk 1]
```diff
@@ -32,6 +32,14 @@ export default {
...
       options.nativeOn = listeners;
     }
+    /**
+     * Supress Vue warnings of .native modifiers
+     * Reference comment and issue #9999 on Vuetify
+     * https://github.com/vuetifyjs/vuetify/issues/9999#issuecomment-579434865
+     */
+    if (tag === 'button') {
+      options.nativeOn = null;
+    }
```
Comment (AustinGil) [inline src/components/VAction/VAction.vue:42]: Do we need this? Isn't nativeOn already undefined for a button? It's only added to `RouterLink` above

File: src/components/VBreadcrumbs/VBreadcrumbs.vue [hunk 2]
```diff
@@ -38,11 +39,11 @@ export default {
   },
 
   computed: {
-    routeBreadcrumbs() {
-      if (this.breadcrumbs.length) {
+    routeBreadcrumbs({ $route }) {
+      if (this.breadcrumbs['length']) {
```
Comment (AustinGil) [inline src/components/VBreadcrumbs/VBreadcrumbs.vue:43]: Was this just an auto formatting change?

File: src/components/VBreadcrumbs/VBreadcrumbs.vue [hunk 3]
```diff
@@ -38,11 +39,11 @@ export default {
...
         return this.breadcrumbs;
       }
-      if (this.$route.fullPath === '/dashboard') {
+      if ($route['fullPath'] === '/dashboard') {
```
Comment (AustinGil) [inline src/components/VBreadcrumbs/VBreadcrumbs.vue:46]: Same question as above

File: src/components/VAction/VAction.vue (hunk 1 above)
Comment (ontoneio) [reply src/components/VAction/VAction.vue:42]: You are correct that it is undefined on the button. However Vue has a nasty annoying `[warn]` label `error` that is only alleviated by explicitly removing it. It just makes for just a bit of nicer DX and doesn't affect the overall functionality of the component.

I reference this:
`Supress Vue warnings of .native modifiers`
     * Reference comment and issue #9999 on Vuetify
     * [Issue #9999-Comment](https://github.com/vuetifyjs/vuetify/issues/9999#issuecomment-579434865)

File: src/components/VBreadcrumbs/VBreadcrumbs.vue (hunk 2 above)
Comment (ontoneio) [reply src/components/VBreadcrumbs/VBreadcrumbs.vue:43]: Correct

Comment (ontoneio) [reply src/components/VBreadcrumbs/VBreadcrumbs.vue:43]: Partially correct, to be more specific, Vetur, was screaming at me and seemed to like this format.

File: src/components/VBreadcrumbs/VBreadcrumbs.vue (hunk 3 above)
Comment (ontoneio) [reply src/components/VBreadcrumbs/VBreadcrumbs.vue:46]: See above

**Comment by ontoneio** [comment]
@AustinGil  any additional feedback here? Should I roll this back?

**Comment by AustinGil** [comment]
Hey, sorry. Don't roll it back. It's just a lot and Im running into issues getting my local dev up and running.

**Review by AustinGil — APPROVED** [review]

**Comment by AustinGil** [comment]
Merged! Thanks :)

**Pull request merged**